.github/workflows/ci.yml

```diff
@@ -20,7 +20,9 @@ jobs:
           enable-stack: true
           stack-version: '2.7'
       - name: Install dependencies
-        run: sudo apt-get install -y libxml2-utils libgl1-mesa-dev libglu1-mesa-dev freeglut3-dev libpulse-dev libblas-dev liblapack-dev libasound2-dev
+        run: |
+          sudo apt-get update
```

Comment:
This is a good idea independently of this PR.

Reply:
Yes, this is needed independently of this PR. Without it it can't install the linux packages because the server addresses have changed or something like that.